Dockerfile

```diff
@@ -1,15 +1,15 @@
-FROM mcr.microsoft.com/dotnet/sdk:7.0.100-alpine3.16-amd64 AS build
+FROM mcr.microsoft.com/dotnet/sdk:9.0.200-alpine3.21-amd64 AS build
 
 WORKDIR /tmp
 
 # Pre-install packages for offline usage
 RUN dotnet new console && \
-    dotnet add package Microsoft.NET.Test.Sdk -v 17.4.1 && \
-    dotnet add package xunit -v 2.4.2 && \
-    dotnet add package xunit.runner.visualstudio -v 2.4.5
+    dotnet add package Microsoft.NET.Test.Sdk -v 17.12.0 && \
+    dotnet add package xunit -v 2.8.1 && \
+    dotnet add package xunit.runner.visualstudio -v 3.0.1
```

Comment:
I would keep the old package versions too, as otherwise existing solutions won't be able to run in the test runner.